packages/bpk-component-barchart/src/BpkBarchartBars.js

```diff
@@ -42,7 +44,31 @@ const getBarHeight = (
 const isOutlier = (point, { yScaleDataKey, maxYValue }) =>
   point[yScaleDataKey] > maxYValue;
 
-const BpkBarchartBars = props => {
+type Props = {
+  data: Array<any>,
```

Comment:
We typically try to avoid `any` unless absolutely necessary, is there something more specific we can use here?